documentation/latest/gsg/minikube.md

```diff
@@ -0,0 +1,147 @@
+cd flotta-operator
+```
+
+First, we need to install Openshift-router, the ingress that it's supported now:
+
+```shell
+---> make install-router
+kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/router_rbac.yaml
+clusterrole.rbac.authorization.k8s.io/openshift-ingress-router created
+clusterrolebinding.rbac.authorization.k8s.io/openshift-ingress-router created
+clusterrolebinding.rbac.authorization.k8s.io/openshift-ingress-router-auth-delegator created
+namespace/openshift-ingress created
+serviceaccount/ingress-router created
+kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/route_crd.yaml
+customresourcedefinition.apiextensions.k8s.io/routes.route.openshift.io created
+kubectl apply -f https://raw.githubusercontent.com/openshift/router/master/deploy/router.yaml
+deployment.apps/ingress-router created
+```
+
+Now let's install Flotta on the cluster:
+
+```shell
+make deploy IMG=quay.io/project-flotta/flotta-operator:latest TARGET=k8s
+```
+
+A bunch of CRDs are now created, where the definition can be found
+[here](../operations/crd.md):
+
+```
+---> kubectl  api-resources | grep flotta
+edgeconfigs                                    management.project-flotta.io/v1alpha1   true         EdgeConfig
+edgedevices                                    management.project-flotta.io/v1alpha1   true         EdgeDevice
+edgedevicesets                                 management.project-flotta.io/v1alpha1   true         EdgeDeviceSet
+edgedevicesignedrequest           edsr         management.project-flotta.io/v1alpha1   true         EdgeDeviceSignedRequest
+edgeworkloads                                  management.project-flotta.io/v1alpha1   true         EdgeWorkload
+playbookexecutions                             management.project-flotta.io/v1alpha1   true         PlaybookExecution
+--->
+```
+
+At the same time, in the flotta namespace an operator should be running:
+```
+---> kubectl -n flotta get pods
+NAME                                                  READY   STATUS    RESTARTS   AGE
+flotta-operator-controller-manager-5ffc74fd8c-wv9cm   2/2     Running   0          2m1s
+--->
+```
+
+Flotta is now running, and ready to register edgedevices, to register a
+edgedevice we need first to retrieve the install script, for that a Makefile
+target is created for that:
+
+```
+make agent-install-scripts
+```
+
+Now, two scripts are created:
+  - hack/install-agent-dnf.sh: To install on normal Fedora installations
+  - hack/install-agent-rpm-ostree.sh: To install on rpm-ostree devices.
+
+Minikube is  a local vm, so `$ minikube ip` cannot be reached from your
+edgedevice, for sharing the flotta API por to the edgedevice the best way is to
+use kubectl-forward:
+
+```
+$ kubectl port-forward service/flotta-operator-controller-manager -n flotta 8043 --address 0.0.0.0
+```
+
+On the device, with Fedora already installed, we need to execute the following:
```

Comment:
Is that explained somewhere how this scripts landed on the device? If there is a doc explaining how to conifigure the edgedevice, it could be nice to have a link

Reply:
In my list to add a new section, how to prepare a edgedevice ;-)